compair/api/comparison_example.py

```diff
@@ -136,14 +145,14 @@ def post(self, course_uuid, assignment_uuid):
             answer1.practice = True
             new_comparison_example.answer1 = answer1
         else:
-            return {"error": "Comparison examples must have 2 answers"}, 400
+            abort(400, title="Comparison Example Creation Failed", message="Comparison examples must have 2 answers.")
 
         if answer2_uuid:
             answer2 = Answer.get_active_by_uuid_or_404(answer2_uuid)
             answer2.practice = True
             new_comparison_example.answer2 = answer2
         else:
-            return {"error": "Comparison examples must have 2 answers"}, 400
+            abort(400, title="Comparison Example Creation Failed", message="Comparison examples must have 2 answers.")
```

Comment:
"Comparison Example Not Saved" / "Please add two answers to the comparison example and try again."